presto-main/src/main/java/io/prestosql/cost/StatsAndCosts.java

```diff
@@ -18,6 +18,8 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.graph.Traverser;
+import io.prestosql.execution.StageInfo;
+import io.prestosql.sql.planner.PlanFragment;
```

Comment:
> `Move construction of StatsAndCosts from StageInto to StatsAndCosts`

is it only the code being moved, or any changes?
what's the benefit?

Reply:
That method was really specifc to `StatsAndCosts` and not `QueryMonitor`